app/retail/templates/shared/activity-vue.html

```diff
@@ -332,15 +332,15 @@
               <div class="tag token mr-2">
                 <p class="font-smaller-6 mb-0">
                   {% if row.activity_type == 'new_tip' %}
-                    <span class="value_in_token">{{ row.metadata.amount | floatformat:2}} {{row.metadata.token_name}}</span>
+                    <span class="value_in_token">{{ row.metadata.amount.normalize }} {{row.metadata.token_name}}</span>
                   {% elif row.bounty %}
-                    <span class="value_in_token_bounty">{{ row.bounty.value_true | floatformat:2 }} {{row.bounty.token_name}}</span>
+                    <span class="value_in_token_bounty">{{ row.bounty.value_true.normalize }} {{row.bounty.token_name}}</span>
                   {% elif row.value_in_token_disp %}
-                    <span class="value_in_token_disp">{{ row.value_in_token_disp | floatformat:2 }} {{row.token_name}}</span>
+                    <span class="value_in_token_disp">{{ row.value_in_token_disp.normalize }} {{row.token_name}}</span>
                   {% elif row.metadata.value_in_eth %}
-                    <span class="value_in_eth">{{ row.metadata.value_in_eth | floatformat:2 }} ETH</span>
+                    <span class="value_in_eth">{{ row.metadata.value_in_eth.normalize }} ETH</span>
                   {% elif row.metadata.value_in_token and row.metadata.token_name %}
-                    <span class="value_in_token">{{ row.metadata.value_in_token | floatformat:2}} {{row.metadata.token_name}}</span>
+                    <span class="value_in_token">{{ row.metadata.value_in_token.normalize }} {{row.metadata.token_name}}</span>
```

Comment:
Is this file used?

Reply:
doesn't look like it ! nice catch @octavioamu 
can you double check and remove this file @chibie  ?

Reply:
@thelostone-mc @octavioamu should i delete it or revert the changes to it?

Reply:
delete